dags/utils/slack_operator.py

```diff
@@ -87,28 +87,6 @@ def get_central_time_exec_data(context):
     return local_tz.convert(execution_date_timestamp).format("MM/DD/YYYY hh:mm:ss A")
 
 
-def task_fail_slack_alert_critical(context):
```

Comment:
I'm tagging-along this change. This function was unused, and it certainly wasn't hurting anything, but I feel like our general policy is to remove stuff that isn't running. LMK if you want this to stay in, I'm totally cool with that too.